### PR TITLE
Dockerfile, update readme links and updated authors in `cargo.toml`

### PR DESCRIPTION
**Pull request by realaravinth**
- Dockerfile
- docker-compose for quickly trying out Triox
- Updated links in readme to point to this repo
- Added my name to authors list in cargo.tom.
- Added new badges for license, code coverage and the matrix community
- Readme formatting. Please see [my copy](https://github.com/realaravinth/Triox) for the rendered version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,102 +1,157 @@
...
+Since we can't guarantee that no illegal data will be uploaded the demo
+server runs in read-only mode.
 
 **[triox-demo.aaron-erhardt.de](https://triox-demo.aaron-erhardt.de)**
 
+## Self-hosted:
+
+1. Clone the repository
+
+`bash git clone https://github.com/Trioxidation/triox cd triox`
```
Comment (AaronErhardt) [inline README.md:59]: Looks a bit odd to me but I haven't tried it. Wouldn't it be better to write something like this?
```
``bash
git clone https://github.com/Trioxidation/triox && cd triox
``
```

**Review by AaronErhardt — COMMENTED** [review]
Looks great overall

File: README.md (hunk 1 above)
Comment (realaravinth) [reply README.md]: yeah sorry about that, the text wrapping messed up code formatting

File: README.md [hunk 2]
```diff
@@ -1,102 +1,106 @@
...
 
-## Contributing
+## Self-hosted:
 
-Everyone is welcome to contribute to Triox. We are always open for new ideas, features and improvements.
+1. Clone the repository
 
-The easiest way to contribute changes is to fork Triox, and then create a pull request to ask us to pull your changes into our repository. You can find a list of good first issues [here](https://github.com/aaronerhardt/triox/labels/good%20first%20issue).
+```bash
+git clone https://github.com/Trioxidation/triox cd triox
```
Comment (AaronErhardt) [inline README.md:60]: I think there needs to be a && before the cd as well.

**Review by AaronErhardt — COMMENTED** [review]
The README looks a lot better now. I think there should be a paragraph about configuration and the bare metal part needs to be rewritten but that can be done later.

**Comment by realaravinth** [comment]
I'll add build instructions in a separate PR

**Pull request merged**